packages/types/src/crypto.ts

```diff
@@ -4,6 +4,8 @@ export type SourceData = string | ArrayBuffer | ArrayBufferView;
  * An object that provides a hash of data provided in chunks to `update`. The
  * hash may be performed incrementally as chunks are received or all at once
  * when the hash is finalized, depending on the underlying implementation.
+ *
+ * @deprecated use {@link Checksum}
```

Comment:
Add `import { Checksum } from "./checksum";` at the top of the file so that the type would be linked